pimd/pim_nb_config.c

```diff
@@ -1882,11 +1882,20 @@ int lib_interface_pim_hello_holdtime_destroy(struct nb_cb_destroy_args *args)
  */
 int lib_interface_pim_bfd_create(struct nb_cb_create_args *args)
 {
+	struct interface *ifp;
+	struct pim_interface *pim_ifp;
+
 	switch (args->event) {
 	case NB_EV_VALIDATE:
 	case NB_EV_PREPARE:
 	case NB_EV_ABORT:
+		/* NOTHING */
+		break;
 	case NB_EV_APPLY:
+		ifp = nb_running_get_entry(args->dnode->parent, NULL, true);
+		pim_ifp = ifp->info;
+		pim_ifp->bfd_config.enabled = true;
+		pim_bfd_reg_dereg_all_nbr(ifp, true);
```

Comment:
Should not be called here. It should be done only in `apply_finish`.

Reply:
Fixed